src/sdk/account/account.service.ts

```diff
@@ -265,6 +266,41 @@ export class AccountService extends Service {
     return result;
   }
 
+  async getAccountInvestments(account: string, ChainId: number, apps?: string[], provider?: string): Promise<AccountInvestments> {
+    const { apiService } = this.services;
+
+    const { result } = await apiService.query<{
+      result: AccountInvestments;
+    }>(
+      gql`
+        query($ChainId: Int!, $account: String!, $apps: [String!], $provider: String) {
+          result: accountInvestments(chainId: $ChainId, account: $account, apps: $apps, provider: $provider) {
+            items {
+              name
+              network
+              position
+              balance
+              logoURI
+            }
+          }
+        }
+      `,
+      {
+        variables: {
+          account,
+          ChainId,
```

Comment:
if `chainId` isn't pass use the value from the session variable.

Reply:
@ch4r10t33r It is already there in sdk file